src/index.html

```diff
@@ -5,9 +5,365 @@
+            Miami Beach, FL 33162
+          </a>
+        </div>
+        <div class="photos">
+          <img
+            src="images/photos/3.jpg"
+            alt="Edition Residence hotel with USA flag"
+            class="gallery__img"
+          >
+          <img
+            src="images/photos/4.jpg"
+            alt="Edition Residence hotel"
+            class="gallery__img"
+          >
+        </div>
+        <div class="gallery__text">
+          Edition Residence /
+          <a href="https://www.google.com/maps/place/%D0%9C%D0%B0%D1%8F%D0%BC%D1%96-%D0%91%D1%96%D1%87,+%D0%A4%D0%BB%D0%BE%D1%80%D0%B8%D0%B4%D0%B0+33139,+%D0%A1%D0%BF%D0%BE%D0%BB%D1%83%D1%87%D0%B5%D0%BD%D1%96+%D0%A8%D1%82%D0%B0%D1%82%D0%B8+%D0%90%D0%BC%D0%B5%D1%80%D0%B8%D0%BA%D0%B8/@25.7819398,-80.1904184,13z/data=!3m1!4b1!4m5!3m4!1s0x88d9b4661cbb1ae5:0x557bd4f5339da98e!8m2!3d25.783312!4d-80.1484407" class="link">
+            Miami Beach, FL 33139
+          </a>
+        </div>
+        <div class="photos">
+          <img
+            src="images/photos/5.jpg"
+            alt="Faena House hotel"
+            class="gallery__img gallery__img--small-left"
+          >
+          <img
+            src="images/photos/6.jpg"
+            alt="Miami beach landscape"
+            class="gallery__img gallery__img--small-left"
+          >
+        </div>
+        <div class="gallery__text">
+          Faena House /
+          <a href="https://www.google.com/maps/place/%D0%9C%D0%B0%D1%8F%D0%BC%D1%96-%D0%91%D1%96%D1%87,+%D0%A4%D0%BB%D0%BE%D1%80%D0%B8%D0%B4%D0%B0+33140,+%D0%A1%D0%BF%D0%BE%D0%BB%D1%83%D1%87%D0%B5%D0%BD%D1%96+%D0%A8%D1%82%D0%B0%D1%82%D0%B8+%D0%90%D0%BC%D0%B5%D1%80%D0%B8%D0%BA%D0%B8/@25.8231588,-80.1828289,13z/data=!3m1!4b1!4m5!3m4!1s0x88d9b36dc0de02e1:0x57754e79a3d46a91!8m2!3d25.8176795!4d-80.1372757" class="link">
+            Miami Beach, FL 33140
+          </a>
+        </div>
+        <button class="gallery__button button">
+          View all
+        </button>
+      </div>
+    </section>
+
+    <section class="container">
+      <h2 class="heading container__heading" id="consult">
+        Consultation
+        <span class="heading__crown">
+        </span>
+      </h2>
+      <div class="consultation">
+        <div class="consultation__text-block">
+          <h3 class="consultation__title">
+            How long have<br>you do yourself a gift?
+          </h3>
+          <p class="consultation__text">The procedural change mezzo forte starts izoritmichesky pickup at these moments stop LA Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
```

Comment:
```suggestion
          <p class="consultation__text">
            The procedural change mezzo forte starts izoritmichesky pickup at these moments stop LA Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
```

Будь ласка, зверни увагу на форматування. Якщо переносиш на декілька рядків, то перенось все, включаючи текст після відкриваючого тега. І так всюди